File: sofd-core-ad-integration/src/main/java/dk/sofd/organization/ad/service/model/Person.java
package dk.sofd.organization.ad.service.model;

import java.time.LocalDate;
import java.util.Set;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@ToString
@JsonInclude(JsonInclude.Include.NON_NULL)
@EqualsAndHashCode
public class Person {
    private String uuid;
    private String master;
    private boolean deleted;
    private String cpr;
    private String firstname;
    private String surname;
    private String chosenName;
    
    @JsonFormat(pattern = "yyyy-MM-dd")
    private LocalDate firstEmploymentDate;
    
    @JsonFormat(pattern = "yyyy-MM-dd")
    private LocalDate anniversaryDate;
    
    private Set<User> users;
    private Set<Affiliation> affiliations;
	private Set<Phone> phones;
}

File: sofd-core-safety-net/Services/Sofd/Model/Person.cs
using System;
using System.Collections.Generic;

namespace sofd_core_safety_net.Services.Sofd.Model
{
    public class Person
    {
        public string Uuid { get; set; }
        public string Cpr { get; set; }
        public bool Deleted { get; set; }
        public string Master { get; set; }
        public string Firstname { get; set; }
        public string Surname { get; set; }
        public string ChosenName { get; set; }
        public DateTime? FirstEmploymentDate { get; set; }
        public Post RegisteredPostAddress { get; set; }
        public List<Affiliation> Affiliations { get; set; }
        public List<User> Users { get; set; }
        public List<Phone> Phones { get; set; }
    }
}
